src/transformers/models/videomae/feature_extraction_videomae.py

```diff
@@ -81,10 +81,10 @@ def resize_video(self, video, size, resample="bilinear"):
     def crop_video(self, video, size):
         return [self.center_crop(frame, size) for frame in video]
 
-    def normalize_video(self, video, mean, std):
+    def normalize_video(self, video, mean, std, rescale=None, channel_first=True):
         # video can be a list of PIL images, list of NumPy arrays or list of PyTorch tensors
         # first: convert to list of NumPy arrays
-        video = [self.to_numpy_array(frame) for frame in video]
+        video = [self.to_numpy_array(frame, rescale=rescale, channel_first=channel_first) for frame in video]
```

Comment:
When normalizing shouldn't `rescale` **always** be True or does it make a difference? The function `normalize_video` would IMO be easier to understand / read when `rescale` is not an argument but instead we copy-paste this part of the code:

```
image = image.astype(np.float32) / 255.0
```

directly into the function.
Also isn't `video` here already a numpy array so do we have to call `self.to_numpy_array` at all?

Reply:
> When normalizing shouldn't rescale always be True or does it make a difference?

Good question. 

It's made me realise a case when it might not be in `__call__`: an input numpy/torch.tensor image with values rescaled between 0-1 and `do_resize=False`, `do_crop=False`. 

In general it's not always True. The previous behaviour was that the image is rescaled if the input is a PIL image, but not rescaled if it's a numpy array or torch tensor. 

As we convert to `np.ndarray` before `normalize` we need to make sure that:
1. rescaling still happens if `do_normalize=True` and the image was converted from PIL -> numpy 
2. rescaling still happens if `do_normalize=True` and the image has values between 0-255
3. rescaling doesn't happen if `do_normalize` is False
4. `normalize` keeps its previous default behaviour

It's missing an explicit check for 1. & 2. I'll add it now.

Reply:
> Also isn't video here already a numpy array so do we have to call self.to_numpy_array at all?

This was for consistency with other vision feature extractors. They can take PIL images as input and will convert them to numpy arrays (as well as rescale and transpose their axes). For them it's needed for backwards compatibility as the method is public. For this model's feature extractor, we could remove as it's not been released yet. It would remove redundancy but would depart from other normalize methods. What do you think is best?

Reply:
Ah ok if it's still necessary to not have it rescale by default then I think it makes sense to leave as is! 

Just more generally I guess it'd be nice to move scaling and channel re-ordering out of `to_numpy_array` - but that's probs better in another PR :-)